File: java/array-2/sum13.java
/* Return the sum of the numbers in the array, returning 0 for an empty 
 * array. Except the number 13 is very unlucky, so it does not count and 
 * numbers that come immediately after a 13 also do not count.
 */
public int sum13(int[] nums) {
    int sum = 0;
    int i = 0;
      
    while(i < nums.length) {
        if(nums[i] == 13) {
            i += 2;
        } else {
            sum += nums[i];
            i++;
        }
    }
                                          
    return sum;
}

File: python/list-2/sum13.py
# Return the sum of the numbers in the array, returning 0 for an empty array. 
# Except the number 13 is very unlucky, so it does not count and numbers that 
# come immediately after a 13 also do not count.
def sum13(nums):
    s = 0
    i = 0
    while i < len(nums):
        if nums[i] == 13:
            i += 1
        else:
            s += nums[i]
            
        i += 1
        
    return s
